client/state/selectors/is-directly-uninitialized.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Tells whether the Directly RTM widget has started initialization
+ *
+ * @see lib/directly for more about Directly
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Boolean}        Whether the widget is waiting to be initialized
+ */
+
+/**
+ * Internal dependencies
+ */
+import { STATUS_UNINITIALIZED } from 'state/help/directly/constants';
+
+export default function isDirectlyUninitialized( state ) {
+	return state.help.directly.status === STATUS_UNINITIALIZED;
```

Comment:
I'm going to go ahead and merge this as-is since it's not fundamentally broken and I'll add your suggestion as a nice-to-have/future improvement.